### DISCUSSION TITLE
Force https not working

### DISCUSSION THREAD

**Discussion by YxLee611** [quasarframework/quasar#12115] [CLI - SSR mode]
Hi, i tried to change quasar.conf > devServer > https: true
but my website still able to access http instead of https, any idea to fix it?

**Comment by MilosPaunovic** [comment]
It should be:
```js
devServer: {
  server: {
    type: 'https',
  }
}
```

[https://quasar.dev/quasar-cli/quasar-conf-js#property-devserver](https://quasar.dev/quasar-cli/quasar-conf-js#property-devserver)
